README.md

```diff
@@ -5,34 +5,52 @@ This is an example implementation to query https://lebensmittelwarnung.api.bund.
 
 Build the Docker image with:
 ```shell
-docker-compose build
+docker compose build
 ```
 
 Enter the container with:
 ```shell
-docker-compose run --rm development sh
+docker compose run --rm development sh
 ```
 
 Start up the webserver with:
 ```Shell
-docker-compose up development
+docker compose up development
 ```
 
 Run tests - Option 1:
 ```Shell
-docker-compose build #optional
-docker-compose run --rm development sh #enter container
+docker compose build #optional
+docker compose run --rm development sh #enter container
 rspec
 ```
 
 Run tests - Option 2:
 ```Shell
-docker-compose build #optional
-docker-compose run --rm development rspec
+docker compose build #optional
+docker compose run --rm development rspec
 ```
 
 Exit container
 ```Shell
 exit
 ctrl + d
-```
\ No newline at end of file
+```
+
+Prettify git log
+```Shell
+git log --all --decorate --oneline --graph
+```
+
+Debug with breakpoint:
+    ```ruby
+    require 'pry';  binding.pry
+    ```
+    Find Docker image name:
+    ```Shell
+    docker ps
+    ```
+    Attach to image:
+    ```Shell
+    docker attach dockerImageName
+    ```
```

Comment:
Too much indentation
~~~suggestion
```ruby
require 'pry';  binding.pry
```
Find Docker image name:
```Shell
docker ps
```
Attach to image:
```Shell
docker attach dockerImageName
```
~~~